### PR TITLE
fixture, products view, products view test

### PR DESCRIPTION
**Pull request by mccordgh**
# Description
Added the test for products view, and the products view itself.
Added a fixture to populate DB so that I could test the product view working live.

# Related Ticket(s)
#6 

## Testing
[x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x] I certify that all existing tests pass

## Deploy Notes

```sh
git pull
git checkout mm-product-view
python manage.py makemigrations initial_site
python manage.py migrate
python manage.py loaddata populate_db.json
```

## Steps to Test
From project directory:
* python manage.py test initial_site

### CODE REVIEW

File: bangazonsite/initial_site/tests.py [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
+        """
+        Testing if we can list all products on a view
+        """
+        ball = models.Product("1", "1", 2, "Baseball", 33.33, 444)
+        ball.save()
+        
+        client = Client()
+        response = client.get(reverse('products'))
+        
+        self.assertQuerysetEqual(response.context['object_list'], ['<Product: Baseball>']
```
Comment (nchemsak) [inline bangazonsite/initial_site/tests.py]: add ")" at the end of line 29

**Review by nchemsak — CHANGES_REQUESTED** [review]
add ")" at the end of line 29 on tests.py

**Comment by Dmart331** [comment]
:wow: 👍 :1st_place_medal:

**Review by nchemsak — APPROVED** [review]
👍 🕹

**Comment by Dmart331** [comment]
😮 👍

**Pull request merged**